src/components/molecules/MapDialog.tsx

```diff
@@ -38,27 +39,29 @@ const useStyles = makeStyles((theme: Theme) =>
   }),
 );
 
-const MapDialog: FC<IProps> = ({ section, open, onClose, location, onLocationChange, onSearch }) => {
+const MapDialog: FC<IProps> = ({ section, open, onClose, location, roadNumber, onLocationChange, onSearch }) => {
   const classes = useStyles();
   const { t } = useTranslation();
 
   return (
-    <Dialog isShowing={open} onClose={onClose} maxWidth='lg' fullWidth>
+    <Dialog isShowing={open} onClose={onClose} maxWidth="lg" fullWidth>
       <Box className={classes.wrapper}>
         <Box className={classes.dialogHeader}>
           <Typography.Title1>{t('mapDialog.searchSection')}</Typography.Title1>
         </Box>
-        <Box display='flex' flexDirection='column' height='75vh'>
-          <Box display='contents'>
+        <Box display="flex" flexDirection="column" height="75vh">
+          <Box display="contents">
             <LocationSelect location={location} onLocationChange={onLocationChange} />
           </Box>
           <div className={classes.chosenSection}>
             <Typography.Body1 bold>{t('mapDialog.chosenSegment')}</Typography.Body1>
-            <Typography.Body1>{section}</Typography.Body1>
+            <Typography.Body1>{roadNumber && section &&` ${t('mapDialog.road')} ${roadNumber} - ${section}`}</Typography.Body1>
```

Comment:
maybe it is better  not to mount an empty element  if `roadNumber` and `section` are falsy ( something like
`{roadNumber && section &&  <Typography.Body1>... /<Typography.Body1>`)

Reply:
```suggestion
            {roadNumber && section && <Typography.Body1>... /<Typography.Body1>
```